src/widget/form/settings/generalsettings.ui

```diff
@@ -377,7 +377,7 @@ instead of system taskbar.</string>
           <item>
            <widget class="QCheckBox" name="cbSeparateWindow">
             <property name="text">
-             <string>Open chats in separate window.</string>
+             <string>Use multiple windows</string>
```

Comment:
it does not say, it is not possible. Same issue you would have with the original., according to you it suggests you cannot open a detached window, but you can.
This option changes the default behaviour. 
I don't think, making the working of that option unnecessarily long would be a good thing to do.
So you would be mor ehappy with something like this: "Automatically use multiple windows (detached mode)"